### DISCUSSION TITLE
Prototipos de pantalla

### DISCUSSION THREAD

**Discussion by SofiaYYQ** [Arquisoft/dede_es5a#45] [Ideas]
Holaa
Otra cosa que podemos ir haciendo son los prototipos, ya que mirando la documentación, descubrí que se puede ponerlos en el apartado 8 crossing-cutting: 😄
![image](https://user-images.githubusercontent.com/79652551/155017550-20e76e9b-5de0-4d5c-9e8e-31496b64e310.png)

De momento se me ocurren las siguientes partes:
- Pantalla del login
- Pantalla del registro
- El logo de la aplicación
- Pantalla principal: Cuando logeamos correctamente a la aplicación.
- Pantalla para visualizar un producto en detalles
- Pantalla para ver todos los pedidos
- Pantalla para ver los detalles de un pedido

**Comment by UO250825** [comment]
He estado buscado alguna herramienta para prototipar:

- [Pencil](https://pencil.evolus.vn/)
- [Quant](https://www.quant-ux.com/#/features.html)
- [Penpot](https://penpot.app/)

Creo que estas tres son gratuitas y open-source

¿Cuál os parece mejor? @SofiaYYQ @mariaurrutia97

**Reply by SofiaYYQ** [reply]
Encontré también otra herramienta, gratuita y open-source, es más bien para los mocks.

- [Draw.io](https://www.google.com.hk/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwimsq2s_pf2AhXMO-wKHVslAosQFnoECAgQAQ&url=https%3A%2F%2Fwww.draw.io%2F&usg=AOvVaw0q79gxfa0gEc3wjqpw3nu9)

![image](https://user-images.githubusercontent.com/79652551/155493394-4014d8ae-8003-43c5-94b4-4ab5d19c41d5.png)

**Reply by mariaurrutia97** [reply]
La de Penpot parece que está bastante bien

**Reply by diegoglezsu** [reply]
draw.io la he usado para los diagramas del contexto de negocio y negocio y esta bien.
